config/shell_log.py

```diff
@@ -5,8 +5,15 @@
 [X] failure_text
 plain_text
 '''
+
```

Comment:
this file can be removed.

use a custom logging formatter for colored console output.